tests/components/grid.test.ts

```diff
@@ -1362,16 +1327,7 @@ describe("Events on Grid update viewport correctly", () => {
 
   test("Move selection vertically (top to bottom) through pane division resets the scroll", async () => {
     freezeRows(model, 3);
-    document.activeElement!.dispatchEvent(
-      // scroll completely to the right
-      new WheelEvent("wheel", {
-        deltaX: 0,
-        deltaY: 4 * DEFAULT_CELL_HEIGHT,
-        shiftKey: false,
-        deltaMode: 0,
-        bubbles: true,
-      })
-    );
+    triggerWheelEvent(".o-grid input", { deltaY: 4 * DEFAULT_CELL_HEIGHT });
```

Comment:
`triggerWheelEvent(document.activeElement!, ...`
to keep the same test as before